docs/archive/project-json-archive.md

```diff
@@ -33,7 +33,7 @@ Works with `project.json`.
 
 ## Constraining package versions with restore
 
-*Originally in [Package restore](../consume-packages/package-restore.md#constraining-package-versions-with-restore).*
+*Originally in [Package restore](../consume-packages/package-restore.md#constrain-package-versions-with-restore).*
```

Comment:
@v-thepet why was this change made?

Reply:
@karann-msft - @Mikejo5000 reviewed the PR above. I made the changes he requested, and he approved. 
I changed "constraining" to "constrain" in package-restore,md because Microsoft Docs style dictates use of action verbs like "Constrain" instead of gerund forms like "Constraining." The change is slowly being made in legacy content so this is one of the updates we always make. The bookmark links then need to change to prevent the broken link.
Note that I've opened an alternative PR #1490 that only contains the two bullets about Azure DevOps and one typo fix. Thanks for your review and approval of either one.